examples/hello_triangle_c/CMakeLists.txt

```diff
@@ -3,13 +3,20 @@ cmake_minimum_required(VERSION 3.11b)
 project(hello_triangle)
 
 set(TARGET_NAME hello_triangle)
-
+set(feature-native)
 add_executable(hello_triangle main.c)
 
 if(MSVC)
     add_definitions(-DWGPU_TARGET=WGPU_TARGET_WINDOWS)
     target_compile_options(${TARGET_NAME} PRIVATE /W4)
     set(GLFW_LIBRARY glfw3)
+
+    target_link_libraries(${TARGET_NAME} userenv ws2_32 Dwmapi)
+    
+    if ("${feature-native}" STREQUAL "dx12")
+        target_link_libraries(${TARGET_NAME} d3dcompiler D3D12 DXGI)
```

Comment:
I'd slightly prefer to have a variable for `BACKEND_LIBRARIES` and add it to the end of the existing `target_link_libraries` – like we do with `OS_LIBRARIES`

Reply:
I do agree with your previous comment: to set the `OS_LIBRARIES` variable with these values. So the script will stay clean